core/sdk-core/src/test/java/software/amazon/awssdk/core/internal/http/pipeline/stages/SigningStageTest.java

```diff
@@ -66,7 +66,8 @@
 public class SigningStageTest {
 
     private static final int TEST_TIME_OFFSET = 17;
-    private static final SignerProperty<String> SIGNER_PROPERTY = SignerProperty.create(String.class, "SigningStageTest.key");
+    private static final SignerProperty<String> SIGNER_PROPERTY =
+        SignerProperty.create(SigningStageTest.class, "SigningStageTest.key");
```

Comment:
nit: the value SigningStageTest.key was probably used to avoid the issue with uniqueness. now with namespace, even when we add uniqueness, the prefix in the name wouldn't be necessary. it's fine either ways, though.